pkg/runner/runner.go

```diff
@@ -79,7 +79,7 @@ func Run(cfg *Config) error {
 		tput.InitWatcher(cfg.ThroughputInterval)
 	}
 
-	if cfg.TracerCfg.JaegerCollectorEndpoint != "" {
+	if cfg.TracerCfg.SamplingRatio > 0 {
```

Comment:
hmm, not sure how much I like this. Enabling tracing would now require setting both an endpoint and a sampling ration by the user. I think I'd prefer a sampling ratio of 1.0 by default and enabling tracing if either the jaeger or the Otel collector endpoints are set.

Reply:
Gotcha, valid point. Will update.